rnd/autogpt_server/autogpt_server/app.py

```diff
@@ -0,0 +1,11 @@
+from multiprocessing import Process
+
+from autogpt_server.agent_api import start_server
+from autogpt_server.agent_executor import start_executor
+from autogpt_server.data import ExecutionQueue
+
+if __name__ == "__main__":
+    queue = ExecutionQueue()
+    executor_process = Process(target=start_executor, args=(5, queue))
+    executor_process.start()
+    start_server(queue)
```

Comment:
FastAPI utilizes 1 process - 1 threaded application , so the the executor need to live as a separate process out of FastAPI and running in parallel. 

Currently, the server (initiated by `start_server`) will communicate the executor (initiated by `startExecutor) by pushing tasks through the `ExecutionQueue` and the pool of threads from the executor will consume it (this is where the task will be replaced by execution of the network of AutoGPT agents.